### ISSUE TITLE
Confusing documentation on use of import statement

### ISSUE THREAD

**Issue by m1sta** [dataformsjs/dataformsjs#22]
I'm trying to migrate from babel to jsxLoader and I'm very confused about my ability to use import statements. The homepage indicates in the first example code block that it is possible by using ```data-type='module'```, but when I try the following I get an error about using the import statement outside a module.

```
<!doctype html>
<html lang="en">
    <body>
        <div id="root" dir="auto"></div>
        
        <script src="https://unpkg.com/react@16.13.1/umd/react.production.min.js" crossorigin="anonymous"></script>
        <script src="https://unpkg.com/react-dom@16.13.1/umd/react-dom.production.min.js" crossorigin="anonymous"></script>
        <script src="https://cdn.jsdelivr.net/npm/dataformsjs@4.0.1/js/react/jsxLoader.min.js"></script>  

        <script type="text/babel" data-type="module">
            import Gallery from 'gallery'
        </script>

    </body>
</html>
```
I'm similarly confused as to whether there is a method to allow use of ```export``` within a component.

**Comment by ConradSollitt** [comment]
Hi,

Sorry for the delay. Which example from the homepage are you referring to?

`import` is not shown on the main site at https://dataformsjs.com/en/ and for examples on https://awesome-web-react.js.org/ you have to view the source to see `import` statements.

See this topic for the main jsxLoader documentation:
https://github.com/dataformsjs/dataformsjs/blob/master/docs/jsx-loader.md#handling-node-importexport-statements-and-browser-exportsrequire

And also the topic on Code Splitting: https://github.com/dataformsjs/dataformsjs/blob/master/docs/jsx-loader.md#code-splitting-%EF%B8%8F

Basically, jsxLoader if you had a standard JS file then `import Component from './file.js'` would work, but to handle JSX code or components then a few lines of config are typically needed. Many examples exist on this site if you view the source code: https://awesome-web-react.js.org/

Is this a popular third-party library that you are trying to include or something that you wrote yourself?

**Comment by m1sta** [comment]
I was look at this "example" from the homepage 

```
<!--
    If a script uses `import` or requires other features on available with
    JavaScript Modules you can specify [data-type="module"] so that the compiled
    script will be added to the page as <script type="module">.

    [data-type="module"] is also supported by Babel Standalone.
-->
<script type="text/babel" data-type="module">
    import { object } from 'library'
</script>
```

I'm just trying to get one jsx file I wrote myself to import another jsx file I wrote myself, but I want to do everything in the browser. I'm currently using babel standalone, which works fine, but I'd prefer to use jsxLoader.

**Comment by m1sta** [comment]
@ConradSollitt any advice?

**Comment by ConradSollitt** [comment]
Hi @m1sta 

Thanks for the demo! That helps and I was able to quickly re-create it on the DataFormsJS playground site https://dataformsjs.com/en/playground

I added working code to your demo:
https://gist.github.com/m1sta/63c9525297092df8ac78c85d31477c52?permalink_comment_id=4499648#gistcomment-4499648

For the documentation I updated the `import` comments because they were not clear that it only imports regular JavaScript files. For dynamic JSX I published a custom component `<LazyLoad>` that can be used.

jsxLoader is designed to load files in the order specified so I'm a little surprised its not working  but I do I recall this issue from a earlier development where the main code should be inline and then modules can be in any order. It see it's not in the documentation so I will review comments and other demos this week and see if I can get it working since it works with Babel Standalone. At the very least I will update the docs in more detail for this.

Here is the largest React demo published on the main site. If you view the source (either in the browser or 2nd link below) you can see that it loads the first page and then all other pages for the SPA come from other files. Some are loaded when the page first loads and some are using `<LazyLoad>`.

https://dataformsjs.com/examples/places-demo-react.htm#/en/

https://github.com/dataformsjs/dataformsjs/blob/master/examples/places-demo-react.htm
